orb/orb.yml

```diff
@@ -257,7 +257,7 @@ commands:
       - run:
```

Comment:
Seen as we are downloading and installing yq in more than one place - should we extract this step and put it into an yaml anchor or a command?

Reply:
Check the latest commit, i am not sure the yaml is correct though